apps/customer/src/app/(tabs)/main/_components/MainPageClient.tsx

```
@@ -1,20 +1,167 @@
+      <AddressSearchBottomSheet
+        open={isAddressSheetOpen}
+        onClose={handleCloseAddressSheet}
+        onSelectAddress={handleSelectAddress}
+      />
     </main>
   );
+}
+
+function mapServerTagToMainCategory(tag: StoreTag): MainCategory {
+  switch (tag) {
+    case "CAFE":
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)